psl/builtin-connectors/src/cockroach_datamodel_connector.rs

```diff
@@ -133,10 +133,14 @@ impl Connector for CockroachDatamodelConnector {
         63
     }
 
-    fn referential_actions(&self, relation_mode: &RelationMode) -> BitFlags<ReferentialAction> {
+    fn referential_actions(&self, _relation_mode: &RelationMode) -> BitFlags<ReferentialAction> {
```

Comment:
I think the `relation_mode` argument on this method can go away, right?

Reply:
Yes, I had highlighted that in a comment above :)

https://github.com/prisma/prisma-engines/pull/3274#discussion_r991722903

Reply:
Ah yes, let's remove it then.